app/controllers/ontologies_metadata_curator_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OntologiesMetadataCuratorController < ApplicationController
+    layout :determine_layout
+
+
+
+    def result
+        @ontologies_ids = params[:ontology][:ontologyId].drop(1)
+        @metadata_sel = params[:search][:metadata].drop(1)
+        @ontologies = []
+        @submissions = []
+        @ontologies_ids.each do |data|
+            @ontologies << LinkedData::Client::Models::Ontology.all.detect {|e| e.acronym == data}
```

Comment:
Not optimized code.

You call  `LinkedData::Client::Models::Ontology.all` inside your for each loop, which means if you have 10 ontologies_ids you will fetch all the ontologies each time.